v3/src/models/shared/shared-case-metadata.ts

```diff
@@ -87,9 +109,27 @@ export const SharedCaseMetadata = SharedModel
     },
     showAllAttributes() {
       self.hidden.clear()
+    },
+    setColor(attrId: string, color: string, high: boolean) {
```

Comment:
How about
```suggestion
    setColor(attrId: string, color: string, selector: "low" | "high") {
```
and then clients can use meaningful strings rather than `true`/`false`?